install.md

```diff
@@ -572,6 +572,7 @@ and export it as a variable, like so:
 ```shell
 rm /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
 
+echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /"| sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
```

Comment:
Hey @tvorogme, can you provide a reproducer for this kind of issue? Usually the kubic repositories should not affected by changes in https://github.com/cri-o/packaging, so I'm wondering what we're missing here.

Reply:
You can start debian 11 container and run without change with OS Debian_11, on any version. It will not install cri-o-runc from apt (no such package). I add this echo from other tutorials and do `apt install cri-o-runc` successfully

Reply:
@haircommander do we need `crio-o-runc` for the kubic repos as well? I'm confused.

Reply:
only for the legacy obs packages. I think we can have this but we are moving away from this packaging scheme so we'll soon no longer use cri-o-runc